sdk/openai/openai/samples-dev/getEmbeddings.ts

```diff
@@ -8,34 +8,32 @@
  * @azsdk-weight 100
  */
```

Comment:
does it make sense to rename this file and getImages away from the "get_" prefix from the previous API naming? e.g. getImages.js to images.js?

Reply:
good eyes, done!